<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnasayfaWidget</class>
 <widget class="QWidget" name="AnasayfaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2b2b2b; color: white; font-family: 'Segoe UI', sans-serif;</string>
  </property>
  <widget class="QFrame" name="cpu_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>320</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#cpu_frame {
    background-color: #2b2e38;
    border-radius: 15px;
    border: 1px solid #3c404d;
}
QLabel { background-color: transparent; color: #ffffff; }</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="cpu_baslik_label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;CPU&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="cpu_model_label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:400; color:#bec3cd;&quot;&gt;Intel Core i9-13900K&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="cpu_grafik_tutucu" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <widget class="QLabel" name="placeholder_cpu_grafik">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>301</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #666; font-size: 10pt;</string>
       </property>
       <property name="text">
        <string>Grafik Alanı (QtCharts veya Custom Widget)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="cekirdek_label">
        <property name="text">
         <string>Çekirdek:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="cpu_cores_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="cpu_hiz_label">
        <property name="text">
         <string>Hız:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="cpu_clock_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="cpu_cache_label">
        <property name="text">
         <string>Önbellek:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="cpu_cache_value_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="cpu_price_label">
        <property name="text">
         <string>Fiyat:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="cprice_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="cpu_score_label_2">
        <property name="text">
         <string>OrtalamaPuan:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="cpu_score_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="gpu_frame">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>320</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#gpu_frame {
    background-color: #2b2e38;
    border-radius: 15px;
    border: 1px solid #3c404d;
}
QLabel { background-color: transparent; color: #ffffff; }</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="gpu_baslik_label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;GPU&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="gpu_model_label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:400; color:#bec3cd;&quot;&gt;NVIDIA GeForce RTX 4090&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="gpu_grafik_tutucu" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <widget class="QLabel" name="placeholder_gpu_grafik">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>301</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #666; font-size: 10pt;</string>
       </property>
       <property name="text">
        <string>Grafik Alanı (QtCharts veya Custom Widget)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QLabel" name="vram_label">
        <property name="text">
         <string>VRAM:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="gpu_vram_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="gpu_hiz_label">
        <property name="text">
         <string>Hız:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="gpu_clock_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="gpu_memtype_label">
        <property name="text">
         <string>Bellek Tipi:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="gpu_memtype_value_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="gpu_price_label">
        <property name="text">
         <string>Fiyat:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="g_price_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="gpu_score_label_2">
        <property name="text">
         <string>OrtalamaPuan:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="gpu_score_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="ram_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>320</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#ram_frame {
    background-color: #2b2e38;
    border-radius: 15px;
    border: 1px solid #3c404d;
}
QLabel { background-color: transparent; color: #ffffff; }</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="ram_baslik_label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;RAM&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ram_model_label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:400; color:#bec3cd;&quot;&gt;GPU&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="ram_grafik_tutucu" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <widget class="QLabel" name="placeholder_ram_grafik">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>301</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #666; font-size: 10pt;</string>
       </property>
       <property name="text">
        <string>Grafik Alanı (QtCharts veya Custom Widget)</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="0">
       <widget class="QLabel" name="kapasite_label">
        <property name="text">
         <string>Kapasite:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="ram_gb_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="ram_hiz_label">
        <property name="text">
         <string>Hız:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="ram_clock_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="ram_type_label">
        <property name="text">
         <string>Türü:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="ram_type_value_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="ram_price_label">
        <property name="text">
         <string>Fiyat:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="r_price_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="ram_score_label">
        <property name="text">
         <string>OrtalamaPuan:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="ram_score_label_2">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="laptop_info_frame">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>320</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#laptop_info_frame {
    background-color: #2b2e38;
    border-radius: 15px;
    border: 1px solid #3c404d;
}
QLabel { background-color: transparent; color: #ffffff; }</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_laptop">
    <item>
     <widget class="QLabel" name="laptop_baslik_label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;Cihaz Bilgisi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="laptop_model_label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:400; color:#bec3cd;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="laptop_price_label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#2ecc71;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_laptop">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="kullanici_bilgisi_frame">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>320</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#kullanici_bilgisi_frame {
    background-color: #2b2e38;
    border-radius: 15px;
    border: 1px solid #3c404d;
}
QLabel { background-color: transparent; color: #ffffff; }</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="kullanici_baslik_label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;Kullanıcı Bilgisi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="kullanici_icon_label">
        <property name="minimumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string></string>
        </property>
        <property name="pixmap">
         <pixmap resource="../../../resources.qrc">:/Assets/user.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="kullanici_adi_label">
           <property name="text">
            <string>Misafir Kullanıcı</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 16px; font-weight: bold;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="global_ranking_label">
           <property name="text">
            <string>Global Sıralama: -</string>
           </property>
           <property name="styleSheet">
            <string notr="true">font-size: 12px; color: #aaa;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      <item>
       <widget class="QPushButton" name="hesapSilButon">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: transparent;
    color: #ff4d4d;
    border: 1px solid #ff4d4d;
    border-radius: 5px;
    padding: 5px 10px;
    font-size: 12px;
}
QPushButton:hover {
    background-color: #ff4d4d;
    color: white;
}</string>
        </property>
        <property name="text">
         <string>Hesabımı Sil</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="sistem_ozeti_frame">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>150</y>
     <width>320</width>
     <height>380</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#sistem_ozeti_frame {
    background-color: #2b2e38;
    border-radius: 15px;
    border: 1px solid #3c404d;
}
QLabel { background-color: transparent; color: #ffffff; }</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="ozet_baslik_label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;Sistem Özeti&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="general_score_container" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_score">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="gauge_chart_container" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="general_score_status_label">
         <property name="text">
          <string>Genel Performans Skoru</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 10pt; color: #aaa;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mini_cards_layout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QFrame" name="mini_cpu_frame">
        <property name="styleSheet">
         <string notr="true">background-color: transparent; border-radius: 5px;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_mini_cpu">
         <item>
          <widget class="QLabel" name="mini_cpu_title">
           <property name="text">
            <string>CPU</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #3498db; font-weight: bold;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="mini_cpu_score">
           <property name="text">
            <string>-</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white; font-size: 12pt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="mini_gpu_frame">
        <property name="styleSheet">
         <string notr="true">background-color: transparent; border-radius: 5px;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_mini_gpu">
         <item>
          <widget class="QLabel" name="mini_gpu_title">
           <property name="text">
            <string>GPU</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #e74c3c; font-weight: bold;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="mini_gpu_score">
           <property name="text">
            <string>-</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white; font-size: 12pt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="mini_ram_frame">
        <property name="styleSheet">
         <string notr="true">background-color: transparent; border-radius: 5px;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_mini_ram">
         <item>
          <widget class="QLabel" name="mini_ram_title">
           <property name="text">
            <string>RAM</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #f1c40f; font-weight: bold;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="mini_ram_score">
           <property name="text">
            <string>-</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white; font-size: 12pt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>

   </layout>
  </widget>
  <widget class="QPushButton" name="sistemibilgilerinitarabuton">
   <property name="geometry">
    <rect>
     <x>671</x>
     <y>540</y>
     <width>311</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #363a45;
    color: white;
    border: 1px solid #3c404d;
    border-radius: 8px;
    font-weight: bold;
    font-size: 13px;
}
QPushButton:hover {
    background-color: #4facfe;
    border: 1px solid #4facfe;
}</string>
   </property>
   <property name="text">
    <string>SİSTEM BİLGİLERİNİ GÜNCELLE / TARA</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
